docs/topics/caching-mission-intro.adoc

```diff
@@ -0,0 +1,32 @@
+Mission proficiency level: xref:proficiency_advanced[*{proficiency-advanced}*].
+
+The {name-mission-caching} mission demonstrates how to use a cache to increase the response time of applications.
+
+This mission shows you how to:
+
+* Deploy a cache to OpenShift.
+* Use a cache within an application.
+
+.About Caching
+Caching allows you to store information and access it for a given period of time.
+An advantage of caching is that a cache can provide access to information faster or more reliably than repeatedly calling the original service.
+A disadvantage of caching is that the cached information is not up to date but that can be mitigated by setting an _expiration_ or TTL (time to live) on each value stored in the cache.
+
+For example, assume you have two applications: _service1_ and _service2_. 
+If _service1_ depends on a value from _service2_ that infrequently changes, _service1_ could cache the value from _service2_ for a period of time. 
```

Comment:
Current wording is ambiguous: is it the value from service2 that infrequently changes, or is it the service2 itself that infrequently changes?